src/test/regress/expected/temp.out

```diff
@@ -170,3 +170,19 @@ select * from whereami;
 -- you can invoke a temp function explicitly, though
 -- select pg_temp.whoami();
 drop table public.whereami;
+create temp table temptest (row integer, count integer);
```

Comment:
I think it would be better to have the exact scenario that failed:
```
CREATE TABLE pg_temp.temptest ...
SELECT avg(pg_temp.temptest.count) ... 
```
Do you think we should cover all three cases (db.table.col, table.col, col)?